examples/lighting-app/nrfconnect/main/AppTask.cpp

```diff
@@ -360,6 +366,8 @@ void AppTask::ActionCompleted(LightingManager::Action_t aAction)
     {
         LOG_INF("Turn Off Action has been completed");
     }
+
```

Comment:
This code will be reached even if the action is coming from the data model itself, right?  Do we really want to re-enter the data model at that point with the attribute write?

Reply:
Good point, I forgot about checking if action was triggered by pushing button manually. I fixed it.